extension/src/experiments/columns/collect/util.ts

```diff
@@ -14,9 +17,7 @@ export const limitAncestorDepth = (ancestors: string[], sep: string) => {
     from what we want the final layer count to be.
   */
   const collectedLimit =
-    Number(getConfigValue(ConfigKey.EXP_TABLE_HEAD_MAX_LAYERS)) ||
-    INITIAL_TABLE_HEAD_MAX_LAYERS
-
+    Number(getConfigValue(ConfigKey.EXP_TABLE_HEAD_MAX_LAYERS)) || limit
```

Comment:
`getConfigValue` always defaults to 5 but since the function could return undefined according to typescript, I couldn't take the `|| limit` part off without receiving a TS error. 

Trying to adjust `getConfigValue` also just lead to other different kinds of errors 🤔

Reply:
@julieg18 In my version the getConfigValue doesn't return 5 by default. I couldn't think of a way to remove the || limit part either.

But this worked for me:

In config.ts:
```
export const getConfigValue = <T = string, D = string>(
  key: ConfigKey,
  defaultValue?: D | T
): T =>
  workspace.getConfiguration().get(key, defaultValue ?? '') as unknown as T
```

And in util.ts:
```
  const collectedLimit = Number(
    getConfigValue(
      ConfigKey.EXP_TABLE_HEAD_MAX_LAYERS,
      limit
    )
  )
```

Reply:
> @julieg18 In my version the getConfigValue doesn't return 5 by default.

When you're running the tests (it should be returning `undefined` in the tests since it's just a mock)? Or when you're in production/development?

Reply:
No, sorry, I meant in general based only on the compiler results. I didn't run the tests specifically

Reply:
> But this worked for me:

Works great, thanks!